src/global-styles.js

```diff
@@ -4,11 +4,11 @@ import { createGlobalStyle } from 'styled-components';
  * Use direct css imports for FCE components
  * This will save some bundle size
  */
-import '@redhat-cloud-services/frontend-components/components/ConditionalFilter.css';
-import '@redhat-cloud-services/frontend-components/components/EmptyTable.css';
-import '@redhat-cloud-services/frontend-components/components/PrimaryToolbar.css';
-import '@redhat-cloud-services/frontend-components/components/Section.css';
-import '@redhat-cloud-services/frontend-components/components/TableToolbar.css';
+import '@redhat-cloud-services/frontend-components/ConditionalFilter/conditional-filter.css';
+import '@redhat-cloud-services/frontend-components/EmptyTable/EmptyTable.css';
+import '@redhat-cloud-services/frontend-components/PrimaryToolbar/primary-toolbar.css';
+import '@redhat-cloud-services/frontend-components/Section/section.css';
+import '@redhat-cloud-services/frontend-components/TableToolbar/TableToolbar.css';
```

Comment:
Its no longer required to import any of these CSS files. They will be injected when the corresponding component is used.